src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1427,9 +1428,10 @@ public function testInheritEnvDisabled()
      *
      * @return Process
      */
-    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60, array $options = array())
+    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60)
     {
-        $process = new Process($commandline, $cwd, $env, $input, $timeout, $options);
+        $process = new Process($commandline, $cwd, $env, $input, $timeout);
+        $process->inheritEnvironmentVariables(true, false);
```

Comment:
but this means that people are then required to call this deprecated method to have the non-deprecated behavior, and then they have to pass the internal second argument to avoid the deprecation warning (and their code won't still be ready for 4.0, if the method is meant to be removed in 4.0, breaking the upgrade path)